docs/running-node-aws-terraform-ansible/deployment.md

```diff
@@ -0,0 +1,251 @@
+# DCL Deployment on AWS using Terraform and Ansible
+
+## Overview
+This document describes all necessary steps to deploy a new DCL network on AWS cloud in accordance with this [design document](../deployment-design-aws.md).
+
+## Prerequisites
+Make sure you have all [prerequisites](./prerequisites.md) set up
+## Terraform and Ansible Configuration
+#### 1. Set up an AWS user for use with Terraform
+    
+Create credentials file [`~/.aws/credentials`] with the following content:
+```text
+[default]
+aws_access_key_id = <access_key_id_here>
+aws_secret_access_key = <secret_access_key_here>
+```
+> **_Note:_** Your account must have enough privileges to manage all AWS resources required by terraform
+
+#### 2. Disable host key checking for Ansible (to avoid host key checking when Ansible connects to AWS instances using ssh)
+
+Create Ansible configuration file [`~/.ansible.cfg`] with the following content:
+```
+ANSIBLE_HOST_KEY_CHECKING=False
+```
+
+## Deployment Configuration
+
+#### 1. Configure AWS infrastructure parameters
+[`deployment/terraform/aws/terraform.tfvars`]
+    
+#### AWS Regions:
+```
+region_1 = "us-west-1"
+region_2 = "us-east-2"
+```
+- Selects two regions where nodes will be created
+
+#### (Genesis) Validator:
+```hcl
+validator_config = {
+    instance_type = "t3.medium"
+    is_genesis    = true
+}
+```
+- Set `is_genesis = false` to deploy just a validator node (not genesis). This option requires:
+  - Putting `genesis.json` file of an existing network to the following path before the step [run-ansible](#4-run-ansible)
+    ```text
+    deployment/persistent_chains/<chain-id>/genesis.json
+    ```
+    where `<chain-id>` is the chain id of a network being joined.
+     - For `testnet-2.0` the genesis file is already in place
+        ```text
+        deployment/persistent_chains/testnet-2.0/genesis.json
+        ```
+  - Manually adding the validator to the network (see [making node a validator](../running-node-ansible/vn.md#make-your-node-a-validator-target-machine)) after the step [run-ansible](#4-run-ansible)
+  - Manually set persistent_peers string in validator config
```

Comment:
But if private sentries are used, then persistent peers should be set for them, not for VN.